code/core/scripts/prep.ts

```diff
@@ -59,7 +59,7 @@ async function run() {
   console.log(isWatch ? 'Watching...' : 'Bundling...');
 
   const files = measure(generateSourceFiles);
-  const packageJson = measure(() => generatePackageJsonFile(entries));
+  const packageJson = measure(() => generatePackageJsonFile(entries.concat(bundles)));
```

Comment:
I can not really place this change.

Reply:
Sorry, it's too long ago. I don't remember why I changed this.

Reply:
I see now, it generates the package.json export entries for the bundles.